debian/changelog

```diff
@@ -0,0 +1,5 @@
+pantos-service-node (1.8.0) UNRELEASED; urgency=medium
+
+  * pantos-service-node: Initial debian packaging
+
+ -- Pantos GmbH <contact@pantos.io> Thu, 13 Jun 2024 14:00:15 +0000
```

Comment:
Missing new line at the end.

Reply:
We can't add it in the changelog or dpkg will complain :(